src/sentry/tagstore/snuba/backend.py

```diff
@@ -157,21 +162,50 @@ def __get_tag_keys_for_projects(
         limit=1000,
         keys=None,
         include_values_seen=True,
+        use_cache=True,
```

Comment:
Since this is going to change the current behavior sensibly now that you aligned the cache keys to five minutes intervals, it is really important to do a survey of who is using the endpoint (the http one) and double check if there are cases where we should start by passing False.
Also, if this is called by a public api we need to inform @BYK before changing the behavior.
The alternative is having default = False and start by enabling it in the use cases we need it most. This implies the cache parameter needs to be on the http endpoint as well.

Reply:
I defaulted this to false instead, and only enabled it for the OrganizationTags endpoint.